google-cloud-spanner/src/main/java/com/google/cloud/spanner/Options.java

```diff
@@ -159,6 +160,13 @@ public static TransactionOption optimisticLock() {
     return OPTIMISTIC_LOCK_OPTION;
   }
 
+  /**
+   * Specifying this instructs the transaction to request {@link IsolationLevel} from the backend.
+   */
+  public static TransactionOption isolationLevelOption(IsolationLevel isolationLevel) {
```

Comment:
Sorry, last nit on this: Can we rename this to the following to be consistent with the other options

```suggestion
  public static TransactionOption isolationLevel(IsolationLevel isolationLevel) {
```